src/charm.py

```diff
@@ -507,7 +507,7 @@ def _resource_reqs_from_config(self) -> ResourceRequirements:
         return adjust_resource_requirements(limits, requests, adhere_to_requests=True)
 
     def _on_k8s_patch_failed(self, event: K8sResourcePatchFailedEvent):
-        self.unit.status = BlockedStatus(event.message)
+        self.unit.status = BlockedStatus(str(event.message))
```

Comment:
Just curious why the ignore here?

Reply:
There's no ignore here :) I guess it was meant for the other line